### ISSUE TITLE
4K MP4 from GoPro not playing

### ISSUE THREAD

**Issue by Gegele1** [Piwigo/piwigo-videojs#182]
Hi,  Thank you for this plugin very useful for adding HD clips or video to my piwigo photo server.   It works very well with my HD videos coming from Canon or Sony cameras, but I cannot get to play video clips from my 4K GoPro, using whatever player version.  The codec description of the clips is the following:

Flux 0
		Codec : MPEG-H Part 2/HEVC (H.265)(hevc1)
		Langue: Anglais
		Type: vidéo
		Résolution vidéo: 3840x2160
		Dimensions du tampon: 3840x2160Format décodé:
		Orientation: haut-gauche
		Couleurs primaires: ITU-R BT.709
		Fonction de transfert de couleur: ITU-R BT.709
		Espace colorimétrique: Gamme ITU-R BT.709
Flux 1
		Codec: MPEG AAC Audio (mp4a)
		Langue: Anglais
		Type: Audio
		Canaux: Stéréo
		Fréquence d'échantillonnage: 48000 Hz
		Bits par échantillon: 32

my environment : Piwigo 11.5.0 
Système d'exploitation: FreeBSD
PHP: 7.4.23 [2021-12-02 13:33:32]
MySQL: 5.5.5-10.5.12-MariaDB [2021-12-02 14:33:32]
Bibliothèque graphique: GD 2.3.

Is this an issue or a limitation?  If an issue, what could I do to fix it?

Many thanks again for the work done,

**Comment by schymans** [comment]
I found this issue because I am having the same problem. If it is a limitation, it would be good to provide the maximum number of pixels and frame rate that can be handled by the plugin, so that we can convert accordingly? I can't try it out right now, but I just responded here so that I can find the issue easily and comment after I have gathered more experience. I have used ffmpeg to convert a gopro video and it works well on piwigo, but I don't remember the full command.
